### PR TITLE
GeoJSON FIX

### PR DESCRIPTION
**Pull request by vadekh**
## Related Issue

https://github.com/digitallinguistics/spec/issues/329
## Description

**This should fix all of the broken PRs. However, it will have a conflict with the changes made to the Language-8.4 PR version of `ajv.js`.**
Pulled the GeoJSON schema offline and into the test library. The `ajv.js` file now accesses an offline version of the GeoJSON schema, which is stored in the `test/utilities` library. This will eliminate a dependency on an unchanging URL.
## Changelog

NEW: add `GeoJSON.json` to `test/utilities`
CHANGE: update `getGeoJSONSchema()` function in `ajv.js` to reference offline version of schema
CHANGE: update `getGeoJSONSchema()` function description
CHANGE: update GeoJSON url in `Location` schema

## Pull Request Checklist

- [x] **[Open an issue](https://github.com/digitallinguistics/spec/issues/new)** for the change, if one is not open already. **NOTE:** It is a good idea to wait until your suggested change is approved by a maintainer before writing any code.

- [x] **Fork the repository** and clone it to your computer.

- [x] **Create an issue branch** for the changes you are making. You should only make changes on this branch for the issue you are currently working on.

- [x] **Install [Node.js](https://nodejs.org/en/)**. Use the latest LTS release. If you need to manage multiple versions of Node.js on your computer, install [nvm](https://github.com/creationix/nvm) or [nvm windows](https://github.com/coreybutler/nvm-windows).

- [x] **Install project dependencies** by running `npm install` from the command line in the project root.

- [x] **Update the project documentation** (`README.md`, `CONTRIBUTING.md`, etc.) to reflect the changes you plan to make, if applicable.

- [x] **Write or update (failing) tests** for the changes you are going to make (if needed). Tests only need to be written for issues that cannot be checked using the example data in the schemas.

- [x] **Make changes to the schemas or code.** The schemas are located in `/schemas/yaml`. Do not update the schemas in `schemas/json`, or your changes will be overwritten.

- [x] **Increment the version number** for any schemas you made changes to.

- [x] **Update the `"description"` field** of any schemas / properties that you change.

- [x] **Update examples** in the affected schemas to reflect the changes you made.

- [x] **Get the latest changes from the `master` branch** and resolve any conflicts that arise. Run the tests again.

- [x] **Build the project** (`npm run build`)

  - Generates JSON versions of the schemas
  - Regenerates the project documentation

- [x] **Run the tests** (`npm test`) and fix issues that arise. If you find an issue that is unrelated to the one you are working on, open a new issue for it.

- [x] **Check the project documentation** for your changes

- [x] **Open a pull request** from your issue branch into the `master` branch. See GitHub's advice on [How to write the perfect pull request](https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/).

- [ ] **Wait for a reviewer to provide feedback**

- [ ] **Address any changes requested by the reviewer**. While working on your pull request, you can write `[ci skip]` in each of your commit messages to skip automated testing, until you are ready to test your code.

- [ ] **Wait for your pull request to be merged.**

- [ ] **Delete your issue branch.**

- [ ] **Celebrate!!!** :fireworks: Your work is complete! Thank you for your valuable contribution to DLx!

### CODE REVIEW

File: test/utilities/ajv.js [hunk 1]
```diff
@@ -12,25 +12,13 @@ let schemasLoaded = false;
 // METHODS
 
 /**
- * Retrieves the GeoJSON schema from an external URL and returns it as a JSON object
+ * Retrieves the GeoJSON schema from the GeoJSON.json file
  * @return {Promise}
  */
 function getGeoJSONSchema() {
```
Comment (dwhieb) [inline test/utilities/ajv.js]: The `getGeoJSONSchema` method is actually redundant now that the GeoJSON schema is loaded directly from a JSON file. The `require` statement at the top loads the GeoJSON schema and makes it available to the rest of this script, so you don't have to wait for it asynchronously. You can delete this method entirely, and refer directly to the `GeoJSON` constant instead.

This means you can also update this line: https://github.com/digitallinguistics/spec/pull/330/files#diff-a5509d3e2a64ff6e2ca19860925a96a7R55

Comment (vadekh) [reply test/utilities/ajv.js]: I have removed the method.

**Comment by vadekh** [comment]
Sorry my last few commits were a bit messy. Probably should've all been one commit, but the checks are passing now.

**Comment by dwhieb** [comment]
Not a problem! That's why I used a Squash & Merge commit. It combines each of your commits on this branch into a single commit into the master branch.

https://help.github.com/en/articles/about-pull-request-merges#squash-and-merge-your-pull-request-commits

The important thing is that the master branch history is clean. Since this branch will be deleted after it's merged, it's totally fine if your commit history on this branch is messy.

**Pull request merged**